src/main/webapp/WEB-INF/hw/task2/quotes.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="/src/main/webapp/WEB-INF/hw/task2/quotesStyle.css">
```

Comment:
We could use relative path

Reply:
Early `resources` package was placed in a root of my project. So, I moved package to `task1` directory and now relative path is: `../resources/article_image.png`

Reply:
@DHrytsuk Updated: [7cc84ffc](7cc84ffc8f5d0a8d90cbb644b232f3835fddc3fb)